src/shared/components/common/markdown-textarea.tsx

```diff
@@ -223,6 +223,7 @@ export class MarkdownTextArea extends Component<
                     this.props.maxLength ?? markdownFieldCharacterLimit
                   }
                   placeholder={this.props.placeholder}
+                  spellCheck
```

Comment:
Note that if the user has spellcheck disabled in their browser settings, this will not override it. In other words, this will only work if the user has spellcheck enabled for their browser.

Reply:
Nice, that's simple enough.